libraries/chain/include/eosio/chain/multi_index_includes.hpp

```diff
@@ -13,14 +13,31 @@
 #include <cyberway/chaindb/multi_index.hpp>
 #include <cyberway/chaindb/index_object.hpp>
 
-#define CHAINDB_TAG(_TYPE, _NAME)                   \
-    namespace cyberway { namespace chaindb {        \
-        template<> struct tag<_TYPE> {              \
-            using type = _TYPE;                     \
-            constexpr static uint64_t get_code() {  \
-                return N(_NAME);                    \
-            }                                       \
-        };                                          \
+#define CHAINDB_TABLE_TAG(_TYPE, _NAME, _ACCOUNT)       \
```

Comment:
Request to change order of fields:
`(_TYPE, _ACCOUNT, _NAME)`

It is more readable:
`CHAINDB_TABLE_TAG(eosio::action_history_object, .history, acthistory)`